htdocs/core/lib/functions.lib.php

```diff
@@ -1689,7 +1689,11 @@ function dol_substr($string,$start,$length,$stringencoding='')
 	if (empty($stringencoding)) $stringencoding=$langs->charset_output;
 
 	$ret='';
-	if (function_exists('mb_substr'))
+	if (function_exists('iconv_substr'))
+	{
+		$ret=iconv_substr($string,$start,$length,$stringencoding);
+	}
+	else if (function_exists('mb_substr'))
```

Comment:
That's right i used the else if instead of elseif... In fact this file has 42 else if  and 64 elseif.
I don't really think that's so important here. But i promise that i'll be carefuly for my next devs.